test/Sentry.Maui.Tests/SentryMauiLogcatsTests.cs

```diff
@@ -0,0 +1,72 @@
+using Microsoft.Extensions.Options;
+#if ANDROID
+using Sentry.Android;
+#endif
+
+namespace Sentry.Maui.Tests;
+
+public class SentryMauiLogcatsTests
+{
+    private class Fixture
+    {
+        public MauiAppBuilder Builder { get; }
+        public FakeTransport Transport { get; private set; } = new FakeTransport();
+        public InMemoryDiagnosticLogger Logger { get; private set; } = new InMemoryDiagnosticLogger();
+
+        public Fixture()
+        {
+            var builder = MauiApp.CreateBuilder();
+            builder.Services.AddSingleton(Substitute.For<IApplication>());
+
+            builder.Services.Configure<SentryMauiOptions>(options =>
+            {
+                options.Transport = Transport;
+                options.Dsn = ValidDsn;
+                options.AttachScreenshot = false; //Disable the screenshot attachment to have the logcat as primary attachment
+                options.Debug = true;
+                options.DiagnosticLogger = Logger;
+                options.AutoSessionTracking = false; //Get rid of session envelope for easier Assert
+                options.CacheDirectoryPath = null;   //Do not wrap our FakeTransport with a caching transport
+                options.FlushTimeout = TimeSpan.FromSeconds(10);
+            });
+            Builder = builder;
+        }
+    }
+
+    private readonly Fixture _fixture = new();
+
+#if ANDROID
+    [Fact]
+    public void CaptureException_CheckLogcatType()
+    {
+#if __IOS__
+        Skip.If(true, "Doesn't support logcats");
+#endif
+        var builder = _fixture.Builder.UseSentry(options =>
+        {
+#if ANDROID
+            options.Android.LogCatIntegration = Android.LogCatIntegrationType.All;
+#endif
```

Comment:
```suggestion
            options.Android.LogCatIntegration = Android.LogCatIntegrationType.All;
```

This is covered by the `#if` above.